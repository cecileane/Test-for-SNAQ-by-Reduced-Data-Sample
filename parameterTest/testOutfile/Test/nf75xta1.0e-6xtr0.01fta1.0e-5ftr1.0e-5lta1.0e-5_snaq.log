optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-5,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 1.0e-5.
rootname for files: nf75xta1.0e-6xtr0.01fta1.0e-5ftr1.0e-5lta1.0e-5_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 12157
seed: 12157 for run 1
Sat Oct  1 17:11:46 2016

 BEGIN SNaQ for run 1, seed 12157 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 239 (failures: 75) with -loglik=158.13875 and ht_min=[0.32676,0.04619,0.29504,10.0,1.10715]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 64		 64		 8	 |	 0		 0		 --
mvtarget	 65		 65		 2	 |	 0		 0		 --
chdir		 28		 28		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 80		 33		 1	 |	 3		 3		 --
Total		 239		 192		 13	 |	 3		 3		 3
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 158.13874671891156
(5,(((1,(2,#H7:::0.32675858292839816):10.0):1.1071489705259563,3):0.04619023946645122,(4)#H7:::0.6732414170716019):0.2950369918505515,6);
---------------------
seed: 23093 for run 2
Sat Oct  1 17:11:58 2016

 BEGIN SNaQ for run 2, seed 23093 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 128 (failures: 75) with -loglik=126.74251 and ht_min=[0.2856,1.51213,0.48942,10.0,0.51916]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 3		 3		 1	 |	 NA 		 NA 		 NA 
mvorigin	 34		 34		 1	 |	 0		 0		 --
mvtarget	 33		 33		 1	 |	 0		 0		 --
chdir		 15		 15		 3	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 43		 32		 2	 |	 1		 1		 --
Total		 128		 117		 8	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 126.74250981975389
((3,(5,(6,#H7:::0.28560306684444275):10.0):0.5191606977501724):0.48941901416925476,4,(2,(1)#H7:::0.7143969331555573):1.5121338221068124);
---------------------

Sat Oct  1 17:12:08 2016
MaxNet is ((3,(5,(6,#H7:::0.28560306684444275):10.0):0.5191606977501724):0.48941901416925476,4,(2,(1)#H7:::0.7143969331555573):1.5121338221068124); 
with -loglik 126.74250981975389
