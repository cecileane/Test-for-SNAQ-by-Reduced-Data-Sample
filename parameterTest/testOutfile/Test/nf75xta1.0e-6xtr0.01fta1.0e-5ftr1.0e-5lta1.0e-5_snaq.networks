((3,(5,(6,#H7:::0.28560306684444275):10.0):0.5191606977501724):0.48941901416925476,4,(2,(1)#H7:::0.7143969331555573):1.5121338221068124);, with -loglik 126.74250981975389 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((3,(5,(6)#H7:::0.692561184562346):1.7121400743974786):0.45555399349579284,4,(2,(#H7:::0.3074388154376539,1):9.662046914643831):0.6651683508579859);, with -loglik 128.2341038924265
((5,((4)#H7:::0.6056113962846724,3):0.0288168717420182):0.21657622577304722,6,((#H7:::0.3943886037153276,2):2.8852243230723977,1):1.1902199773684228);, with -loglik 172.3318203769787
((3,(5,(6,((2)#H7:::0.5672544598382223,1):4.7146144096504745):0.1676835687140279):0.026115228816627548):1.7932882950725255,4,#H7:::0.4327455401617777);, with -loglik 192.0585409135528
((3)#H7:::0.5967798463315381,4,(2,(((5,#H7:::0.4032201536684618):9.370235489724815,6):0.9977654329536217,1):0.09404419029315166):0.06987203715248247);, with -loglik 215.12411209750542
((3,(5)#H7:::0.5036846449655588):1.1867508306472632,4,(2,((#H7:::0.4963153550344412,6):4.409651430914708,1):0.04243364866482044):0.20318064888393778);, with -loglik 231.50901217793682
