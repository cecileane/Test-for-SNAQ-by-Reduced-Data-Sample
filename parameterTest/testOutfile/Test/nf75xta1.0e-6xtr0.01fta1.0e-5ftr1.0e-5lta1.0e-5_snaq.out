((3,(5,(6,#H7:::0.28560306684444275):10.0):0.5191606977501724):0.48941901416925476,4,(2,(1)#H7:::0.7143969331555573):1.5121338221068124); -Ploglik = 126.74250981975389
 Dendroscope: ((3,(5,(6,#H7):10.0):0.5191606977501724):0.48941901416925476,4,(2,(1)#H7):1.5121338221068124);
 Elapsed time: 21.922153989 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((3,(5,(6,#H7:::0.28560306684444275):10.0):0.5191606977501724):0.48941901416925476,4,(2,(1)#H7:::0.7143969331555573):1.5121338221068124);, with -loglik 126.74250981975389
 (5,(((1,(2,#H7:::0.32675858292839816):10.0):1.1071489705259563,3):0.04619023946645122,(4)#H7:::0.6732414170716019):0.2950369918505515,6);, with -loglik 158.13874671891156
-------
